dbus/methods.go

```diff
@@ -598,3 +598,43 @@ func unitPath(name string) dbus.ObjectPath {
 func unitName(dpath dbus.ObjectPath) string {
 	return pathBusUnescape(path.Base(string(dpath)))
 }
+
+type JobStatus struct {
+	Id       int
```

Comment:
This is an uint32 according to the dbus method signature.

Reply:
Done